content/en/os/1.15.x/api/settings/dns/_index.markdown

```diff
@@ -0,0 +1,10 @@
++++
+title="dns"
+type="docs"
+toc_hide=true
+description="Settings related to custom DNS settings (`settings.dns.*`)"
++++
+
+Bottlerocket generates the host `/etc/resolv.conf` from `settings.dns.*` values.
```

Comment:
For the `wicked` backend, it's `/etc/resolv.conf`. With `systemd-networkd` and `systemd-resolved`, it's `/run/systemd/resolve/resolv.conf`.